packages/banner/addon/components/nucleus-banner.js

```diff
@@ -46,6 +46,26 @@ class NucleusBanner extends Component {
   */
   @defaultProp
   isFixed = false;
+  
+  /**
+  * Text to be used for localisation of additional banner items message
+  *
+  * @field showMore
+  * @type string
+  * @public
+  */
+ @defaultProp
+ showMore = 'more';
+
+ /**
+  * Text to be used for localisation of System Alerts message
+  *
+  * @field systemAlerts
+  * @type string
+  * @public
+  */
+ @defaultProp
+ systemAlerts = 'System Alerts';
```

Comment:
Ideally banners should follow conventions as `danger`, `warnings`, `information`, and `success`. Any reason why we call this API as `systemAlerts`?

Reply:
When there are multiple banners, they are stacked under a "+ [x] more" button. On clicking the button, we get a list of the banners under the title "System Alerts". This is just a prop to customise that title. :) 

![Screenshot 2020-02-05 at 3 55 05 PM](https://user-images.githubusercontent.com/59554914/73833361-e1f7d600-482f-11ea-94d1-f1b50e9ae12e.png)

Reply:
The title "System Alerts" itself seems to be a part of the design spec.

Reply:
Renamed the props now for clarity.